src/apps/persistence/app_groups.clj

```diff
@@ -34,6 +34,16 @@
                  (fields :id :name :is_public)
                  (where {:id app_group_id}))))
 
+(defn search-app-groups
+  "Searches for app category information by name."
+  [names]
+  (-> (select* [:app_categories :c])
```

Comment:
Keep in mind that public apps these days use ontology metadata for categories, and not these old db categories.

Is this for QA tests?

Reply:
Yeah, this is for QA tests.